### ISSUE TITLE
num_beams arg

### ISSUE THREAD

**Issue by HuskyKingdom** [JeremyLinky/YouTube-VLN#7]
Hi, thanks for your work!

I would like to ask a quick question about CML args. While testing, I was trying to understand the fundamentals behind data loading/evaluation, after a bit of playing around, it appears to me that although arg 'num_beams' is provided as a parameter, no matter how I change it, the model would always predict logits of 30 trajectories for each 'instr_id', is this expecting or have I made any mistakes?

Looking forward to your replay :))
